exercises/luhn/canonical-data.json

```diff
@@ -123,7 +115,23 @@
       "expected": true
     },
     {
-      "description": "strings with non-digits is invalid",
+      "comments": [
+        "Convert non-digits to their ascii values and then offset them by 48 sometimes accidentally declare an invalid string to be valid.",
+        "This test is designed to avoid that solution."
+      ],
+      "description": "using ascii value for doubled non-digit isn't allowed",
```

Comment:
can you double-check me on this one? I think in this case, the `b` is in a **non**-doubled position, and in the latter case the `:` is in a doubled position. So these two test case descriptions should be switched, right?

Or did I get them mixed up